api/_utils.ts

```diff
@@ -729,6 +727,14 @@ export function applyMapFilter<InputType, MapType>(
   }, []);
 }
 
+export function resolveEthersError(err: unknown): string {
+  return sdk.typeguards.isEthersError(err)
+    ? `${err.reason}: ${err.code}`
+    : sdk.typeguards.isError(err)
+    ? err.message
+    : "unknown error";
```

Comment:
I think the code construction here is OK, but this auto-formatting is ++annoying. I'd prefer it to be:
```suggestion
  return sdk.typeguards.isEthersError(err)
    ? `${err.reason}: ${err.code}`
    : sdk.typeguards.isError(err)
      ? err.message
      : "unknown error";
```

Reply:
Have manually overridden it via a prettier directive.